### DISCUSSION TITLE
Is it possible to add additional classes to editor elements?

### DISCUSSION THREAD

**Discussion by CorLynns** [GrapesJS/grapesjs#4403] [Q&A] [answered]
Is it possible to add additional classes to elements in the editor? I'm not talking about components in the canvas.

I want to restrict what editing features certain classes of users can adjust. For example, I don't want beginner users to worry about box-shadows but more experienced users can explore that editing option. So I want to add a class of say "advanced-user" to the box-shadow property so I can create a script that toggle the "advanced-user" class's display so its hidden for user deemed basic but visible for users we've deemed more advanced

**Comment by artf** [comment, marked as answer]
I think it would be easier to handle this manually with JS (eg. on toggle, swap "*-user-mode" classes) and a bit of CSS, eg.
```css
.standard-user-mode .gjs-sm-property__box-shadow {
 display: none;
}
```

**Reply by CorLynns** [reply]
that's how i was going to go about it but I was wondering if it were possible to get in to give an additional class to the specific items so I'd only have to toggle that one class instead calling on each individual item, example .advanced-options instead  .gjs-sm-property__transition, .gjs-sm-property__line-height,  .gjs-sm-property__letter-spacing, AND   .gjs-sm-property__text-shadow
